gcp/modules/couchdb/main.tf

```diff
@@ -2,10 +2,64 @@ terraform {
   backend "gcs" {}
 }
 
+variable "couchdb_prd_replicas" {
+  default = 3
+}
+
+variable "couchdb_dev_replicas" {
+  default = 2
+}
+
+variable "release_namespace" {
+  default = "gpii"
+}
+
+variable "env" {}
 variable "secrets_dir" {}
 variable "values_dir" {}
-variable "release_namespace" {
-	default = "gpii"
+variable "project_id" {}
+variable "serviceaccount_key" {}
+
+provider "google" {
+  project     = "${var.project_id}"
+  credentials = "${var.serviceaccount_key}"
+}
+
+resource "google_compute_disk" "couchdb" {
+  count = "${var.env == "dev" ? var.couchdb_dev_replicas : var.couchdb_prd_replicas}"
+  name  = "couchdb-${count.index}"
+  type  = "pd-ssd"
+  size  = 10
```

Comment:
This is set to 5 on the AWS side. Why did you change it here?

Reply:
You can not create PDs smaller than 10GB for most storage classes in GCP.